src/github/projectKit.ts

```diff
@@ -2,32 +2,60 @@ import { graphql } from "@octokit/graphql";
 
 import { ILogger, IProjectApi, Issue, Repository } from "./types";
 
-interface ProjectData {
-  organization: {
-    projectNext: {
-      id: string;
-      fields: {
-        nodes: {
-          id: string;
-          name: string;
-          settings?: string | null;
-        }[];
-      };
-    };
-  };
+type NodeData = { id: string; title: string };
+
+export const PROJECT_V2_QUERY: string = `
```

Comment:
Any objections to using [graphql-tag](https://www.npmjs.com/package/graphql-tag) on the queries? It'll bring syntax highlighting at least

Reply:
OMG I think this is exactly what I need. Let me try it out!

Reply:
there's also one method, which gives you autocomplete of GQL 
Implemented in this project: https://github.com/paritytech/roadmap/blob/main/getProjects.graphql

But this is pain in ass to set this up :)

Reply:
Yeah, we just discussed to skip it, we already have all the queries written down and [the tests for the queries](https://github.com/paritytech/github-issue-sync/pull/50/files#diff-166bb1d8f11fa6c00b35cd25d25a496c116826c7c4ec40f7de96a2797154c88a) are already validating that the queries can compile and are using the correct types.

If they are modified and become invalid the tests will throw errors like:
- `GraphQLError: Syntax Error: Expected Name, found <EOF>.`
- `GraphQLError: Cannot query field "titles" on type "ProjectV2". Did you mean "title", "fields", or "items"?`

Reply:
sounds good